### PR TITLE
Feature/pcf exchange kit update rel 24 05

### PR DESCRIPTION
**Pull request by wern**
## Description
Update of PCF Exchange Kit to Version Rel. 24.05

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/PCF Exchange Kit/page_software-operation-view.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: Operation View
 description: 'PCF Exchange KIT'
 sidebar_position: 4
 ---
->**Note**
->
-> At the moment there is no open source soultion available implementing PCF exchange. Therefore no central operation information is provided here.
+In the past, the Catena-X ecosystem for PCF data exchange consisted exclusively of commercial applications (so-called COTS apps). For release 24.05, with the Simple Data Exchanger (SDE), for the first time an open source solution is announced to support the CX PCF Use Case Standard (data model, API,...). Further information regarding the SDE (usage, configuration and deployment) are published on GitHub:
```
Comment (danielmiehle) [inline docs-kits/kits/PCF Exchange Kit/page_software-operation-view.md:7]: I would suggest to adjust the sentence to:

"In addition to certified commercial solutions, there is now also an open-source reference implementation available for the first time..."

**Review by danielmiehle — CHANGES_REQUESTED** [review]
Thanks for the KIT update - the SDE for PCF is a big step forward. I only have minor change request.

File: docs-kits/kits/PCF Exchange Kit/page_adoption-view.md [hunk 2]
```diff
@@ -505,12 +496,9 @@ The diagram shown here illustrates the interaction between the PCF KIT and the o
 
 ## STANDARDS
 
-The standards for release 24.03 will be published soon. Our relevant standards can be downloaded from the official [Catena-X Standard Library](https://catena-x.net/de/standard-library):
+The standards this release will be published soon. Our relevant standards can be downloaded from the official [Catena-X Standard Library](https://catena-x.net/de/standard-library):
```
Comment (jSchuetz88) [inline docs-kits/kits/PCF Exchange Kit/page_adoption-view.md:499]: There seems to be something off with the first sentence. Perhaps "The standard **of** this ..."

**Review by jSchuetz88 — APPROVED** [review]
Only minor remarks from my side, looks good to go.

**Comment by danielmiehle** [comment]
@wern: Can you please add the changes requests asap. We would like to merge the PR today. Thank you!

**Comment by wern** [comment]
@danielmiehle Review finding fixed.

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Comment by danielmiehle** [comment]
@wern: Thanks! Can you also fix the conflicting files asap?

**Comment by wern** [comment]
@danielmiehle @HFocken Trying to check locally, but as there seems to be broken line endings in the repo (outside our changes) it may take some time...

**Comment by danielmiehle** [comment]
> @danielmiehle @HFocken Trying to check locally, but as there seem to be broken line endings in the repo (outside our changes) it may take some time...

Thank you for checking!

**Comment by wern** [comment]
@danielmiehle @HFocken I may be too stupid, but I am not able to resolve the merge conflicts without changing stuff outside of our kit (due to broken, checked in line feeds). Please let me know how we want to proceed here.

**Comment by danielmiehle** [comment]
> @danielmiehle @HFocken I may be too stupid, but I am not able to resolve the merge conflicts without changing stuff outside of our kit (due to broken, checked in line feeds). Please let me know how we want to proceed here.

Please, contact johann.schuetz@catena-x.net for further support.
fyi, @jSchuetz88

**Comment by danielmiehle** [comment]
@wern: so we can close this PR, right?

**Pull request closed without merging**